persistence/sql/migrations/sql/20240318143139000000_drop_identity_search_index.postgres.down.sql

```diff
@@ -0,0 +1,4 @@
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
+CREATE EXTENSION IF NOT EXISTS btree_gin;
+
+CREATE INDEX identity_credential_identifiers_nid_identifier_gin ON identity_credential_identifiers USING GIN (nid, identifier gin_trgm_ops);
```

Comment:
Isn't cock,roach missing here?

Reply:
This is just a copy of the previous migration where the index was created. It works as it should on cockroach as well.

Reply:
But the migration specifically only applies to postgres (see file name)

Reply:
Oh I see - maybe we applied it manually on prod?